src/docs/docs/getting-started/aws-cloud9.md

```diff
@@ -2,8 +2,12 @@
 
 [AWS Cloud9](https://aws.amazon.com/cloud9/) is a cloud-based integrated development environment (IDE) that lets you write, run, and debug your code with just a browser.
 
+:::info
+
 **Disclaimer**: The EC2 instance used in this guide is not within AWS's Free-Tier so please see [EC2 Pricing](https://aws.amazon.com/ec2/pricing/on-demand/) to see if you're comfortable with these costs. Cloud9 has a cost-saving setting to help reduce costs by automatically hibernating after 30 minutes of inactivity. Running Docker in development is CPU intensive so these are the EC2 instances I recommend:
 
+:::
+
```

Comment:
Is there a point to have both a word "disclaimer" and the `Info` box? Perhaps remove **Disclaimer:**